### PR TITLE
[www] Various "Creators/Community"…stuffs-and-things

### LINKED ISSUE #8485: Community: Detail page design tweaks
- [x] update the design of the "badges" on Creator pages
foo

- [x] Add "type" information to Creator detail page (agency, company, etc) 
foo 1

(@fk pulled images from Slack)

### PR DESCRIPTION
**Pull request by fk**
An intermediate step PR… 👣 🐤 

* Hide filters (by commenting)
  * „All/People/Agencies/Companies“
  * „For hire“ and „Hiring“
* Don’t query a creator’s sites in `componentDidMount()` but in `render()`
* Move „For Hire/Hiring“ badge to component, touch styles — slightly calmer "For Hire" on the index page, though still not happy  
  image
* Don’t go below 1rem for main copy on small devices
* Drawing inspiration from the mocks for the Site/Starter Showcase detail view meta design to style „Get in touch“, „From“, and „Worked on“ blocks for small screens  
   * doing so, add <MetaTitle>, <MetaSection> components 👣 🐤(WIP)
  image
* Don’t uppercase creator name
  * stick to general gatsbyjs.org main headline style
  * more readable
  * allow brand- or name-specific upper/lowercase usage (e.g. „LekoArts“)
* Move detail page "For hire/Hiring" badge below creator name and GitHub icon next to it
* Don’t underline detail page „Get in touch/From/Worked On“ "meta" block titles
* Remove protocol from „Get in touch“ link text
image
* Break layout into two columns earlier (`presets.Tablet` instead of `Desktop`)  
image
* Output Creator "type" if its either `company` or `agency` (on the detail page):  
image

Fixes #8485, touches on #8484

### CODE REVIEW

File: www/src/templates/template-contributor-page.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import Img from "gatsby-image"
 import Layout from "../components/layout"
 import Container from "../components/container"
 import BlogPostPreviewItem from "../components/blog-post-preview-item"
+import Badge from "../views/community/badge"
```
Comment (calcsam) [inline www/src/templates/template-contributor-page.js]: needed?

File: www/src/views/community/badge.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  hiring: {
+    background: colors.ui.light,
+    color: colors.lilac,
+  },
+  forHire: {
+    background: `#effaef`,
+    color: colors.success,
+  },
+}
```
Comment (calcsam) [inline www/src/views/community/badge.js]: Are we defining styles objects at the bottom of the file? I know we talked about this at some point @jlengstorf but I don't remember. Either way, does not block merge.

Comment (fk) [reply www/src/views/community/badge.js]: I think I adopted it from @kkemple during .com dev, and have been doing it in a couple of places on `www`, too.

**Comment by LeKoArts** [comment]
> slightly calmer "For Hire" on the index page, though still not happy

The green on green has a contrast of 2.51 (according to Chrome DevTools).
It would be happy with `#2b7e2b` for the text.

**Comment by LeKoArts** [comment]
@fk Oh, I missed that the text of the location in the overview also needs to be adjusted:

image

**Comment by berraknil** [comment]
@fk @calcsam 

This is really disappointing as I would be more than happy to make all these changes upon request before my contract was done (literally the day before this post was made) as I waited for feedback for more than a week at a time, and instead I got a 👍 and apparently there were a lot of things people weren't happy with.

I would have appreciated honest feedback above all.

**Pull request merged**